### PR TITLE
python setup.py install

### PR DESCRIPTION
**Pull request by kaushik-rohit**
## What? Why?
Fix # . (Add Reference to the issue if any) fix setup.py. now install can be done using simple python setup.py install

Changes proposed in this pull request:
-  install numy and scipy in beginning if they don't exist

## Checks
- [ ] Single commit and [No merge commits](http://nathanleclaire.com/blog/2014/09/14/dont-be-scared-of-git-rebase/)
- [ ] Make sure you have added required documentation
- [ ] If you have refactored a code, make sure that you have compared the outputs pre and post refactoring. Add both pre and post refaoring screenshots below if the output is visual

## Notify reviewers
@rohangoel96

### CODE REVIEW

**Comment by rohangoel96** [comment]
Since this involves setup.py fixing.
It'll be great to fix https://github.com/prasadtalasila/IRCLogParser/issues/83 as well in the same PR.
Stackoverflow link added for reference.

File: .travis.yml [hunk 1]
```diff
@@ -15,9 +15,8 @@ python:
 # command to install dependencies
 install:
   - pip install -U pip wheel
```
Comment (rohangoel96) [inline .travis.yml:17]: why use wheel now?

File: setup.py [hunk 2]
```diff
@@ -24,31 +37,40 @@
...
         'nltk',
         'plotly',
         'ddt',
         'codeclimate-test-reporter',
         'memory_profiler',
         'snakefood'
     ],
+    extras_require={
+        'dev': [
+            'pyyaml',
```
Comment (rohangoel96) [inline setup.py]: proper indent

File: setup.py [hunk 3]
```diff
@@ -24,31 +37,40 @@
...
-        'numpy',
+    dependency_links = ['https://github.com/rtfd/sphinx_rtd_theme/archive/0.2.4.tar.gz#egg=sphinx_rtd_theme-0.2.4'],
+
+	setup_requires = [
+        'graphviz'],
+
+    install_requires = [
         'networkx',
         'matplotlib',
+        'graphviz',
```
Comment (rohangoel96) [inline setup.py]: repitition with setup_requires

File: setup.py [hunk 4]
```diff
@@ -24,31 +37,40 @@
...
-        'sphinxcontrib-googlechart',
-        'sphinxcontrib-googlemaps',
-        'sphinxcontrib-httpdomain',
-        'sphinxcontrib-slide',
-        'sphinxcontrib.youtube',
         'nltk',
         'plotly',
         'ddt',
         'codeclimate-test-reporter',
         'memory_profiler',
```
Comment (rohangoel96) [inline setup.py]: repitition with dev

File: .travis.yml (hunk 1 above)
Comment (kaushik-rohit) [reply .travis.yml:17]: this is for upgrading pip. we are still using pip.

File: setup.py (hunk 3 above)
Comment (kaushik-rohit) [reply setup.py]: we need to mention setup time dependencies and install time dependencies at both places.

**Pull request merged**